terraform/modules/cluster/eks.tf

```diff
@@ -44,122 +58,77 @@ module "eks_blueprints" {
       type        = "ingress"
       self        = true
     }
-    # Recommended outbound traffic for Node groups
-    egress_all = {
```

Comment:
These rules are now added by default with the EKS module when `node_security_group_enable_recommended_rules` is `true` (which is the default) - https://github.com/terraform-aws-modules/terraform-aws-eks/blob/df1b62548c1d8c7117f4ab45c5b494de64b34cb8/node_groups.tf#L129-L180

The default rules do permit 1025-65535 for cross security group communication, but I suspect that with the sample applications there are additional ports (i.e. 80) required. I wasn't sure what was used so I left the custom rule addition for all cross security group communication open